gno.land/cmd/gnoweb/main.go

```diff
@@ -1,61 +1,157 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"fmt"
+	"net"
 	"net/http"
 	"os"
 	"time"
 
-	// for static files
 	"github.com/gnolang/gno/gno.land/pkg/gnoweb"
 	"github.com/gnolang/gno/gno.land/pkg/log"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
-	// for error types
-	// "github.com/gnolang/gno/tm2/pkg/sdk"               // for baseapp (info, status)
 )
 
+type webCfg struct {
+	chainid    string
+	remote     string
+	remoteHelp string
+	bind       string
```

Comment:
Just curious; why `bind`?

Reply:
It's common practice to use `bind` as a flag name because it literally binds a listening socket to a specific address and port